packages/story-editor/src/components/canvas/editLayerFocusManager/keyBindings.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useKeyDownEffect } from '@googleforcreators/design-system';
+import { useCallback } from '@googleforcreators/react';
+
+function wrapIndex(i, length) {
+  return ((i % length) + length) % length;
```

Comment:
🤔 Does this need to mod-ed this twice? I think this accomplishes the same thing, no?
```suggestion
  return i % length;
```

It feels like `i` is modulo'd down, then un-modulo'd by adding the length to the result, and then re-modulo'd. Lmk if I'm understanding this wrong!

Reply:
Ah good question. 

This is some classic funkiness with js modulo. say we're on the `0` index of the array, and you want to go to the previous element. you'd go to `(0 - 1) % length`. What you'd expect is that you get taken to `length - 1`, but js takes you to `-1`.

Here's some output examples from just messing with the repl in the console:
<img width="208" alt="Screen Shot 2022-04-05 at 11 26 05 AM" src="https://user-images.githubusercontent.com/35983235/161801452-d3b273b5-515b-4645-b399-f42eedac3c3a.png">

Reply:
waaaaa okay wow, this is wild!

Reply:
Luckily we only have step = 1/-1, but in cases of large negative numbers we'd have to add `length` to the value until it's positive?

So strange!

<img width="94" alt="image" src="https://user-images.githubusercontent.com/22185279/161835877-fe8b3bc9-69e5-4fe7-be33-563f6b913df2.png">

🤔 I wonder why it works differently in JS

Reply:
works for all step sizes. Here's a lil step by step of what the possible sets look like as you navigate from `-infinity -> infinity`
```
// assume length === 5 for brevity
[-infinity, infinity]  %  5
[-4, 4 ] + 5
[ 1, 9 ] % 5
[ 0, 4 ]
```

Reply:
Y'all this hurts my brain.

Reply:
saaaame